lms/static/js/utils/navigation.js

```diff
@@ -0,0 +1,73 @@
+var edx = edx || {},
+
+    Navigation = (function() {
+
+        var navigation = {
+
+            init: function() {
+
+                if ($('#accordion').length) {
+
+                    navigation.openAccordion();
+                }
+            },
+
+            openAccordion: function() {
+                $('#open_close_accordion a').click();
```

Comment:
Hmm, it looks like it's present - in a `header` element, which doesn't seem correct, and hidden. I'm not sure how or why this ties into other parts of the LMS. There is `sr` text for "Close" so I'm assuming there are some scenarios where you can manually toggle the accordion closed, but I've never seen this in LMS before. Any ideas? Should I strip it out?